docs/cookbook/admin_panel/menu.md

```diff
@@ -38,3 +46,46 @@ final readonly class MenuBuilder implements MenuBuilderInterface
     }
 }
 ```
+
+### Add submenu items
+
+<div data-full-width="false">
+
+<figure><img src="../../.gitbook/assets/submenu_items.png" alt="Submenu items"></figure>
+
+</div>
+
+Now you can add submenu items:
+
+```php
+#[AsDecorator(decorates: 'sylius_admin_ui.knp.menu_builder')]
```

Comment:
What about adding use here to avoid searching to the wrong class ?

Reply:
Yep, indeed